src/blocks/ProfilePage/types.ts

```diff
@@ -9,11 +10,15 @@ export type IProfileProps = {
         }
     },
     body?: {
-        userData?: Array<Record<string, string>>;
+        userData?: Array<Record<string, unknown>>;
         form?: IFormProps;
     },
     footer?:{
         links: Array<ILinkProps>;
+        buttons: Array<IButtonProps>;
     },
-    navigation?: ILinkProps;
+    navigation?: {
+        link: ILinkProps,
+        button: IButtonProps,
```

Comment:
Эти моменты тоже стоит привести к одному виду
```suggestion
        link: ILinkProps;
        button: IButtonProps;
```